PackageSourceMappingExample/README.md

```diff
@@ -1,9 +1,322 @@
+
+    a. Both top level (directly installed) *and transitive* packages must match defined patterns. There is no requirement that a top level package and its dependencies come from the same source.
+    
+    b. The same ID pattern can be defined on multiple sources, allowing matching package IDs to be restored from any of the feeds that define the pattern. However, this isn't recommended due to the impact on restore predictability (a given package could come from multiple sources).
+
+3. When multiple unique patterns match a package ID, the most specific (longest) match will be preferred. 
+
+    a. Exact package ID patterns always have the highest precedence while the generic `*` always has the lowest precedence. For an example package ID `NuGet.Common`, the following package ID patterns are ordered from highest to lowest precedence: `NuGet.Common`, `NuGet.*`, `*`. 
+
+4. Package Source Mapping settings are applied following [nuget.config precedence rules](https://docs.microsoft.com/nuget/consume-packages/configuring-nuget-behavior#how-settings-are-applied) when multiple `nuget.config` files at various levels (machine-level, user-level, repo-level) are present. 
+
+> Important: When the requested package already exists in the global packages folder, no source look-up will happen and the mappings will be ignored. Declare a [global packages folder for your repo](https://docs.microsoft.com/nuget/reference/nuget-config-file#config-section) to gain the full security benefits of this feature. Work to improve the experience with the default global packages folder in planned for a next iteration.
+
+## Package Source Mapping onboarding
+
+To fully onboard your repository take the following steps:
+
+1.  Declare a new [global packages folder for your repo][9].
+2.  Run [`dotnet list package --include-transitive`][12] to view all top-level and transitive packages in your solution. 
+    *   For .NET framework projects using [`packages.config`][13], the `packages.config` file will have a flat list of all direct and transitive packages.
+3.  Define mappings such that every package ID in your solution - *including transitive packages* - matches a package ID pattern for the target source.
+4.  Run restore to validate that you have configured your mappings correctly. If your mappings don't fully cover every package ID in your solution, the error messages will help you identify the issue.
+5.  When restore succeeds, you are done! Optionally consider: 
+    *   simplifying the configuration to fewer declarations by using broader package ID prefixes or [setting a default source][14] where possible.
+    *   verifying the source each package was restored from by checking the [metadata files in the global packages folder or reviewing the restore logs][15].
+
+## Short example scenarios and results

```

Comment:
Same as earlier, I wonder if we should link rather than duplicate. 

Eventually the docs will be the source of truth.

Reply:
Removed the duplication